config/main.py

```diff
@@ -1591,6 +1591,79 @@ def incremental(file_name):
     command = "acl-loader update incremental {}".format(file_name)
     run_command(command)
 
+
+#
+# 'dropcounter' group ('config dropcounter ...')
+#
+
+@config.group()
+def dropcounter():
+    """Drop counter related configuration tasks"""
+    pass
+
+
+#
+# 'initialize_counter' subcommand ('config dropcounter initialize_counter')
+#
+@dropcounter.command()
+@click.argument("counter_name", type=str, required=True)
+@click.argument("counter_type", type=str, required=True)
+@click.argument("reasons",      type=str, required=True)
+@click.option("-a", "--alias", type=str, help="Alias for this counter")
+@click.option("-g", "--group", type=str, help="Group for this counter")
+@click.option("-d", "--desc",  type=str, help="Description for this counter")
+@click.option('-v', '--verbose', is_flag=True, help="Enable verbose output")
+def initialize_counter(counter_name, alias, group, counter_type, desc, reasons, verbose):
+    """Initialize a new drop counter"""
+    command = "dropconfig -c install -n '{}' -t '{}' -r '{}'".format(counter_name, counter_type, reasons)
+    if alias:
+        command += " -a '{}'".format(alias)
+    if group:
+        command += " -g '{}'".format(group)
+    if desc:
+        command += " -d '{}'".format(desc)
+
+    run_command(command, display_cmd=verbose)
+
+
+#
+# 'delete_counter' subcommand ('config dropcounter delete_counter')
+#
+@dropcounter.command()
+@click.argument("counter_name", type=str, required=True)
+@click.option('-v', '--verbose', is_flag=True, help="Enable verbose output")
+def delete_counter(counter_name, verbose):
```

Comment:
Is there a reason to include the suffix "_counter" here? It seems a bit redundant. Is it to distinguish it from the "_reasons" subcommands commands below?

Reply:
That's right, although with the _reasons suffix I think it might be clear enough without the _counter suffix.

Reply:
That's kinda what I was thinking, as well.